FinSearchUnified/BusinessLogic/ExportFactory.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace FinSearchUnified\BusinessLogic;
+
+use Exception;
+use FinSearchUnified\ShopwareProcess;
+use Shopware\Bundle\PluginInstallerBundle\Service\InstallerService;
+
+class ExportFactory
+{
+    /**
+     * @var InstallerService
+     */
+    private $installerService;
+
+    private $decorators = [
+        'fin_search_unified.article_model_factory',
+        'fin_search_unified.shopware_process'
+    ];
+
+    /**
+     * ExportFactory constructor.
```

Comment:
Personally not a fan of this. It should already be clear that `__contruct` is the constructor for this class. Please remove.